.travis.yml

```diff
@@ -27,6 +27,14 @@ matrix:
           os: linux
           dist: xenial
           env: TOXENV=py37
+        - language: generic
+          os: osx
+          osx_image: xcode9.4
+          env: TOXENV=py27,py36
+        - language: generic
+          os: osx
+          osx_image: xcode10.1
+          env: TOXENV=py27,py37
```

Comment:
Doesn't hurt, but we don't need to run macOS Python 2.7 here and also above. Maybe just `TOXENV=py37` here?